kafka/librdkafka_vendor/README.md

```diff
@@ -1,5 +1,22 @@
 # Bundling prebuilt librdkafka
 
+>
+> **Note to Zendesk Developers:**
+> 
+> Rather than following this process, simply check out the `kafka/librdkafka_vendor` directory
+> on the upstream. For example:
+> ```shell
+> $ git remote add upstream https://github.com/confluentinc/confluent-kafka-go
+> ...
+> $ git fetch --all
+> ... 
+> $ git checkout v1.9.0 -- kafka/librdkafka_vendor
+> ...
+> # still need to regenerate error codes
+> $ make -f mk/Makefile generr
+> ...
+> ```
```

Comment:
Just noting a modified process that we can use here for checking out the updated vendored librdkafka upstream.